src/validators/policeoffenses.js

```diff
@@ -13,30 +10,4 @@ export const validatePoliceOffenses = (data, formType, options = {}) => {
     requireLegalOffenseSentenced: requireLegalOffenseSentenced(formType),
     requireLegalOffenseIncarcerated: requireLegalOffenseIncarcerated(formType),
   }
-
-  const policeOffensesModel = {
-    HasOffenses: { presence: true, hasValue: { validator: hasYesOrNo } },
-    List: (value, attributes) => (
-      checkValue(attributes.HasOffenses, 'Yes')
-        ? {
-          presence: true,
-          accordion: { validator: offense },
-        } : {}
-    ),
-  }
-
-  return validateModel(data, policeOffensesModel, { ...options, ...modelOptions })
-}
```

Comment:
The above is part of the `validatePoliceOffenses` function and shouldn't have been deleted

Reply:
Fixed